framework/meta/src/template.rs

```diff
@@ -48,3 +58,63 @@ pub fn copy_template_to_location(template: &str, location: &Path) {
         .join(template);
     let _ = copy_dir(contract_path, location);
 }
+
+pub fn update_dependencies(template: &str) {
```

Comment:
In future PR: rename to template_crate_name.

We need to explicitly set the new project name too. But not now.